event_session/models/event_event.py

```diff
@@ -102,12 +102,12 @@ def _compute_date_end(self):
             if data.get(rec.id):
                 rec.date_end = data.get(rec.id)
 
-    def _check_seats_limit(self):  # pragma: no cover
+    def _check_seats_availability(self):  # pragma: no cover
         # OVERRIDE to ignore this constraint for event with sessions
         # Seat availability is checked on each session, not here.
         session_records = self.filtered("use_sessions")
         regular_records = self - session_records
-        return super(EventEvent, regular_records)._check_seats_limit()
+        return super(EventEvent, regular_records)._check_seats_availability()
```

Comment:
But you should check the session seats availability somehow, and this makes me think that the tests aren't complete. Please check how this has changed in v16 in core and it for the sessions.

Reply:
Note: For events with sessions, the limit is applied on the `event.session` level. The `event.event` is something like a "template"

Reply:
Thanks @ivantodorovich :slightly_smiling_face: